### PR TITLE
Adding Set/Cleaning Data

### PR DESCRIPTION
**Pull request by Aion2501**
Adding set #158 Flowing Black
Cleaning up the order of objects to match how they are displayed in game.
Adding a more visible start line for future additions.

### CODE REVIEW

File: lib/objdata/suitsets [hunk 1]
```diff
@@ -6,31 +6,45 @@
...
 :leg      = 
 :foot     = 
-:finger   = 
 :shield   = 
-:back     = 
 // Paste below this line.
+// ----------------------
+:Set #158
+:#Flowing Black
+:$Human
```
Comment (cizra) [inline lib/objdata/suitsets:37]: Others use lowercase here. I don't know if it matters, but uniformity would be good too

File: lib/objdata/suitsets [hunk 2]
```diff
@@ -6,31 +6,45 @@
...
-:wrist    = 
 :leg      = 
 :foot     = 
-:finger   = 
 :shield   = 
-:back     = 
 // Paste below this line.
+// ----------------------
+:Set #158
+:#Flowing Black
```
Comment (cizra) [inline lib/objdata/suitsets:36]: These `:` should be removed — see the other suitsets below. Currently, it's commented out.

**Comment by wershlak** [comment]
I'm wondering what happens if someone checks out the code and tries to load this set? I assume it will just spam me with "There is no such object." a bunch?

Can we get a recent sql_data dump committed that includes this set?

**Comment by cizra** [comment]
We can do a new DB dump, that's not a big problem. I don't know any elegant ways to version DB states, though...

**Comment by Aion2501** [comment]
Ah yes I see the comments now. What a nub. And you're right that they're all lowercase in the 'load set' command. I think they should have all been uppercase, but that's lots of work. Uniformity it is!

Lots of suitsets have missing slots. I've seen it return the entire list with a few lines that say "there is no object in this slot" or something to let the immortal know.

**Pull request merged**